lib/panel-utils.js

```diff
@@ -324,6 +318,36 @@ function messageHandler(opts) {
   }
 }
 
+// Opens the video in the original tab, if it is still open, else in a new tab.
+function sendToTab(panel, opts) {
+  const pageUrl = getPageUrl(opts.domain, opts.id, opts.time);
+
+  if (pageUrl) {
+    const tabs = require('sdk/tabs');
+    let targetTab = null;
+    for (let tab of tabs) { // eslint-disable-line prefer-const
+      if (tab.id === opts.tabId) {
+        targetTab = tab;
+        break;
+      }
+    }
+
+    const targetVideoId = getVideoId(targetTab.url);
+    const pageVideoId = getVideoId(pageUrl);
+    if (targetTab !== null && pageVideoId && targetVideoId === pageVideoId) {
+      targetTab.url = pageUrl;
```

Comment:
I'm not sure what the intended behaviour is here. Do we want to simply close the min-vid panel and activate the tab, leaving it up to the user to seek to the desired time and play it? Or do we want to auto-play the video from the exact time the panel was closed? I was assuming the latter, which is why I'm setting the URL here.